infrastructure/main.tf

```diff
@@ -149,6 +154,9 @@ module "citizen-frontend" {
 
     CONTACT_EMAIL = "${data.azurerm_key_vault_secret.staff_email.value}"
 
+    //LaunchDarkly
+    LD_SDK_KEY = "${data.azurerm_key_vault_secret.launch_darkly_sdk_key.value}"
+    LD_OFFLINE_FLAG = "${var.ld_offline_flag}}"
```

Comment:
this shouldn't need to be set as it will will default to on

Reply:
removed